bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -36,7 +38,23 @@ class TelegramWorker(
         actorId: TelegramActorId,
         id: UserId
     ): RemoteUser<UserId> {
-        TODO("Not yet implemented")
+        val actorData: TelegramActorData? = actorProvisionRepository.getActorData(actorId)
+        val token = actorData?.token ?: throw IllegalStateException("No token for actor $actorId")
+
+        val bot: Bot = bot {
+            this.token = token
+        }
+
+        val chatId = ChatId.fromId(id.id)
+
+        val chat = bot.getChat(chatId).get()
```

Comment:
```suggestion
        val chat = withContext(Dispatchers.IO) { bot.getChat(chatId).get() }
```

getChat() is not suspending, so it needs to be moved into IO threads (although currently effect is not seen due to dual-threadness of the framework)